### PR TITLE
Pyzmq 16.0.2

### PR DESCRIPTION
**Pull request by jhein32**
Set of easyconfig files for ZeroMQ 4.2.0 and PyZMQ 16.0.2, incl pre-requisites.  
Supporting 

- foss/2016b and intel/2016b  
- Python bindings for 2.7.12 and 3.5.2.
- URL update for OpenPGM

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb [hunk 1]
```diff
@@ -13,9 +13,11 @@ description = """OpenPGM is an open source implementation of the Pragmatic Gener
 toolchain = {'name': 'intel', 'version': '2016b'}
 toolchainopts = {'pic': True}
 
-source_urls = [GOOGLECODE_SOURCE]
+source_urls = ['https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/openpgm/']
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb]: @jhein32 do you mind changing this in all OpenPGM v5.2.122 easyconfigs?

File: easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb [hunk 2]
```diff
@@ -13,9 +13,11 @@ description = """OpenPGM is an open source implementation of the Pragmatic Gener
...
 sources = ['libpgm-%(version)s.tar.gz']
 
+checksums = ['d7673e9ff6cc33cf42fe9fb9a7bfbffa']
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb]: @jhein32 the checksum too please?

**Comment by jhein32** [comment]
@boegel  Update done as requested.  Please note that the files for goolf 1.4.10, ictce 5.3 + 5.5 locked differently before the change.  Please check/confirm that the current syntax, incl the sources line is ok.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (11 easyconfigs in this PR)
node2691.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/48dc1739d2ef9b461d4ac53c38a7186b for a full test report.

File: easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb (hunk 2 above)
Comment (jhein32) [inline easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb]: Comment: that checksum is what I got on the local clusters after downloading from the google archives.  It is not a published checksum from google.

Comment (boegel) [inline easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb]: checksum checks out on our end, was downloaded a long time ago:

```
$ ls -l /apps/gent/source/o/OpenPGM/libpgm-5.2.122.tar.gz 
-rw-r--r-- 1 vsc40003 geasybuild 1026254 Jan  8  2013 /apps/gent/source/o/OpenPGM/libpgm-5.2.122.tar.gz
$ md5sum !$
md5sum /apps/gent/source/o/OpenPGM/libpgm-5.2.122.tar.gz
d7673e9ff6cc33cf42fe9fb9a7bfbffa  /apps/gent/source/o/OpenPGM/libpgm-5.2.122.tar.gz
```

Thanks @jhein32!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 21 out of 21 (20 easyconfigs in this PR)
node2055.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1c69d232c1aa526da6c48f4f08848ef4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jhein32!

**Pull request merged**